src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -399,4 +462,56 @@ private String getDescription(Object descriptionHolder) {
             return Assert.assertShouldNeverHappen();
         }
     }
+
+    private List<Directive> getDirectives(Object directiveHolder) {
```

Comment:
is this method a copy from somewhere? If not we should add more tests for the different cases